.github/workflows/update-pr-labels.yml

```
@@ -0,0 +1,29 @@
+name: Update PR labels
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  after-merge:
+    if: github.event.pull_request.merged == true
```

```suggestion
    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'status: in discussion')
```

This filters for RFC PRs